### ISSUE TITLE
incorrect coverage after switching nyc to c8

### ISSUE THREAD

**Issue by aladdin-add** [bcoe/c8#411]
* **Version**: node v18.3.0
* **Platform**: macos - Darwin MacBook-Pro-7.local 21.6.0 Darwin Kernel Version 21.6.0: Sun Jun 19 22:37:06 PDT 2022; root:xnu-8020.140.41~9/RELEASE_X86_64 x86_64
* c8: v7.11.3

I was just trying to move nyc to c8 in eslint repo, it seems got a incorrect coverage - makes our ci failing:
```bash
=============================== Coverage summary ===============================
Statements   : 79.23% ( 59457/75041 )
Branches     : 90.51% ( 18070/19964 )
Functions    : 82.99% ( 4666/5622 )
Lines        : 79.23% ( 59457/75041 )
================================================================================
```

you can see the commit: [2fb16aeafec93e4372358dc7ee96295505f796e7](https://github.com/aladdin-add/eslint/pull/6/commits/2fb16aeafec93e4372358dc7ee96295505f796e7)
the gh log: https://github.com/aladdin-add/eslint/runs/7246638296

**Comment by bcoe** [comment]
@aladdin-add if you run `c8` locally, with an `html` report, are you able to see what lines are reported as uncovered (which should be covered)?

**Comment by aladdin-add** [comment]
closing, as seems it has been fixed in v7.12.0.
